### DISCUSSION TITLE
模板预览问题，并且点击从模板新建条目子笔记无反应

### DISCUSSION THREAD

**Discussion by wendribss** [windingwind/zotero-better-notes#847] [Q&A] [answered]
当前安装的最新版本是zotero 7.0.0-beta.54，有的模板标题是[Item]时无法预览，有的模板标题是[Text]可以预览，但是点击从模板新建条目子笔记都无反应，请问是什么原因呢？
![2](https://github.com/windingwind/zotero-better-notes/assets/102166608/d0ef6f22-509e-4a22-aedd-f6b4935f60b5)
![3](https://github.com/windingwind/zotero-better-notes/assets/102166608/327f5422-cfb5-4369-b8aa-553dd30cfeff)

**Comment by windingwind** [comment, marked as answer]
升级插件，模板问题咨询模板作者

**Reply by npj06** [reply]
您好，我目前zotero的版本是6.030，betternotes是版本1.0.4，您之前创建的笔记模板Styling of annotaions with different color之前一直使用良好，最近突然不能使用，和这位提问者的状况比较相似，我复制了template code导入至模板中会提示invalid
![Uploading 截屏2024-01-03 21.19.31.png…]()
想寻求一下帮助，谢谢

**Reply by npj06** [reply]
code来源网址为https://github.com/windingwind/zotero-better-notes/issues/85

**Reply by wendribss** [reply]
好的，谢谢解答，插件已经更新到最新版本了，但是点击从模板新建条目子笔记还是没有反应，我换下模板试试看

**Reply by windingwind** [reply]
#85 已经过时，直接在discussion里搜索相关模板的帖子

**Reply by windingwind** [reply]
另外，社区提供的模板一般只支持最新版（目前是只支持zotero7的beta版本）。

**Reply by wendribss** [reply]
好的，谢谢

**Comment by wutherwalker** [comment]
我也是从模板新建笔记没有任何反应，预览也没显示

**Comment by wutherwalker** [comment]
(3)(+0000002): [Better Notes for Zotero] runTemplate: [Item]论文笔记

(3)(+0000002): [Better Notes for Zotero] runTemplate: [Item]论文笔记

(4)(+0000802): SELECT COALESCE(IA.parentItemID, itemID) AS itemID FROM items LEFT JOIN itemAnnotations IA USING (itemID) WHERE (itemID NOT IN (SELECT itemID FROM deletedItems UNION SELECT itemID FROM itemNotes WHERE parentItemID IS NOT NULL AND parentItemID IN (SELECT itemID FROM deletedItems) UNION SELECT itemID FROM itemAttachments WHERE parentItemID IS NOT NULL AND parentItemID IN (SELECT itemID FROM deletedItems)UNION SELECT itemID FROM itemAnnotations WHERE parentItemID IN (SELECT itemID FROM deletedItems)UNION SELECT itemID FROM itemAnnotations WHERE parentItemID IN (SELECT itemID FROM itemAttachments WHERE parentItemID IN (SELECT itemID FROM deletedItems)))) AND itemID IN (SELECT itemID FROM items WHERE (libraryID=?)) AND itemID IN (SELECT itemID FROM items WHERE (itemTypeID IN (SELECT itemTypeID FROM itemTypesCombined WHERE typeName=?))) [1, 'note']

**Comment by hansir579** [comment]
我也是模版有问题  每次通过模版来创建笔记都显示请选择一个有效的分类
